src/Formatter/PlatesFormatter.php

```diff
@@ -3,10 +3,11 @@
 namespace Equip\Formatter;
 
 use Equip\Adr\PayloadInterface;
+use Equip\Formatter\FormatterInterface;
 use Equip\Formatter\HtmlFormatter;
 use League\Plates\Engine;
 
-class PlatesFormatter extends HtmlFormatter
+class PlatesFormatter extends HtmlFormatter implements FormatterInterface
```

Comment:
I don't see what benefit is gained by pushing the `implements` call to extensions of `HtmlFormatter`. It duplicates code unnecessarily.